core/jobs.py

```diff
@@ -941,9 +945,9 @@ def validate(cls, unused_mapper_spec):
 
 
 class BaseMapReduceJobManagerForContinuousComputations(BaseMapReduceJobManager):
-    """Continuous Computation jobs which runs continuously, are used to perform
+    """Continuous Computation jobs, which runs continuously, are used to perform
     statistical, visualisation or any other real time calculations. These jobs
-    are triggered by events for which a particular job is subscribed to.
+    are used to perform calculations outside of a client's request.
```

Comment:
are used to perform background calculations that take place outside the usual client request/response cycle.

Reply:
done.